tools/c7n_mailer/c7n_mailer/sqs_message_processor.py

```diff
@@ -20,12 +20,11 @@
 import yaml
 import zlib
 
-from address import get_user
```

Comment:
'address' is a bit confusing, since there are so many address, sns, email, etc. made the file name more descriptive